tests/integration_tests/data_steward/utils/bq_test.py

```diff
@@ -60,3 +74,45 @@ def test_define_dataset(self):
         # Post conditions
         self.assertIsInstance(results, bigquery.Dataset)
         self.assertEqual(results.labels, self.label_or_tag)
+
+    def test_update_labels_and_tags(self):
+        # Tests if project_id is given
+        self.assertRaises(TypeError, update_labels_and_tags, self.dataset_id,
+                          self.label_or_tag)
+
+        # Tests if dataset_id is given
+        self.assertRaises(TypeError, update_labels_and_tags, self.project_id,
+                          self.label_or_tag)
+
+        # Tests if no label or tag is given
+        self.assertRaises(TypeError, update_labels_and_tags, self.project_id,
+                          self.dataset_id)
+
+        # Pre-conditions
+        results = define_dataset(self.project_id, self.dataset_id,
+                                 self.description, self.label_or_tag)
+
+        self.client = bigquery.Client()
+        self.dataset = self.client.create_dataset(results)
+
+        update_false_expected = update_labels_and_tags(
+            self.project_id, self.dataset_id, self.overwritten_label_or_tag,
+            self.set_or_update_false)
+
+        update_true_expected = update_labels_and_tags(self.project_id,
+                                                      self.dataset_id,
+                                                      self.updated_label_or_tag,
+                                                      self.set_or_update_true)
+
+        # Post conditions
+        self.assertIsInstance(results, bigquery.Dataset)
+        self.assertEqual(update_true_expected.labels, self.updated_label_or_tag)
+        self.assertEqual(update_false_expected.labels,
+                         self.overwritten_label_or_tag)
+
+    def tearDown(self):
+        # Deletes bar_dataset
+        self.client = bigquery.Client()
+        self.client.delete_dataset(f'{self.project_id}.{self.dataset_id}',
```

Comment:
If this integration test remains, we should also do a `delete_dataset()` in `setUp()` just in case